addons/l10n_id_efaktur/tests/test_l10n_id_efaktur.py

```diff
@@ -277,3 +287,359 @@ def test_efaktur_consume_code(self):
         self.assertEqual(self.efaktur.available, available_code - 1)
         # No error is raised when downloading.
         out_invoice_no_taxes.download_efaktur()
+
+    def test_efaktur_post_multi_invoices(self):
+        """ Test to amke sure if we post 2 invoices at once, each of them have their own
+        separate eTax number"""
+
+        # Create 2 separate invoices
+        out_invoice_1 = self.env["account.move"].create({
+            "move_type": "out_invoice",
+            "partner_id": self.partner_id.id,
+            "invoice_date": "2019-05-01",
+            "date": "2019-05-01",
+            "invoice_line_ids": [
+                Command.create({"name": "line1", "price_unit": 110.0, "tax_ids": self.tax_id.ids}),
+            ],
+            "l10n_id_kode_transaksi": "01",
+        })
+        out_invoice_2 = self.env["account.move"].create({
+            "move_type": "out_invoice",
+            "partner_id": self.partner_id.id,
+            "invoice_date": "2019-05-01",
+            "date": "2019-05-01",
+            "invoice_line_ids": [
+                Command.create({"name": "line1", "price_unit": 120.0, "tax_ids": self.tax_id.ids}),
+            ],
+            "l10n_id_kode_transaksi": "01",
+        })
+        out_invoices = out_invoice_1 + out_invoice_2
```

Comment:
```suggestion
Also valid for other tests; please create in batch.
```suggestion
        out_invoices = self.env["account.move"].create([{
            "move_type": "out_invoice",
            "partner_id": self.partner_id.id,
            "invoice_date": "2019-05-01",
            "date": "2019-05-01",
            "invoice_line_ids": [
                Command.create({"name": "line1", "price_unit": 110.0, "tax_ids": self.tax_id.ids}),
            ],
            "l10n_id_kode_transaksi": "01",
        }, {
            "move_type": "out_invoice",
            "partner_id": self.partner_id.id,
            "invoice_date": "2019-05-01",
            "date": "2019-05-01",
            "invoice_line_ids": [
                Command.create({"name": "line1", "price_unit": 120.0, "tax_ids": self.tax_id.ids}),
            ],
            "l10n_id_kode_transaksi": "01",
        }])
```
```